src/translations/product-list.translations.js

```diff
@@ -129,11 +129,40 @@ export const FILTER_BUTTON_TEXT = {
     value: 'Filter'
   }
 };
-
+export const PRODUCT_NOT_FOUND = {
+  0: {
+    lang: 'uk',
```

Comment:
please change it to ua.

Reply:
I can't fix it now becouse it's configuration of all project. So that it should be change everywhere